### ISSUE TITLE
Log level mismatch in LdapFilterPlugin.java

### ISSUE THREAD

**Issue by saledouble** [oracle/opengrok#2751]
In the example below, line 92 uses a log level of `FINER` in a logging statement that describes a failure that happens under a certain condition. In contrast, line 99 contains an informational log but has the same level of `FINER`. This contrast in severity is pronounced by the high-level exception thrown on line 96. Should the level in line 92 be greater than `FINER`?

https://github.com/oracle/opengrok/blob/9ce52a219b9e5772102d1fdfcfd7c3c672a332b4/plugins/src/opengrok/auth/plugin/LdapFilterPlugin.java#L90-L99

Filed in conjunction with @orenwf and @khatchad.

**Comment by vladak** [comment]
Definitely, the log levels in authorization plugins are not very consistent in general.
